src/_gettsim_tests/test_data/arbeitsl_geld_2/2005/hh_id_7.yaml

```diff
@@ -72,11 +68,11 @@ inputs:
       - 0.0
       - 0.0
       - 0.0
-    eink_st_y_tu:
+    eink_st_y_sn:
       - 0.0
       - 0.0
       - 0.0
-    soli_st_y_tu:
+    soli_st_y_sn:
```

Comment:
I think the test here doesn't make much sense because it assumes that three adults>25 have a Bedarfsgemeinschaft together. Should we get rid of it then?

Reply:
Yes, that does not make any sense. Can't see much of the file on the phone, but if there are no comments in it that should make us reconsider, just delete it.

Reply:
No comments, and it's hard to tell what the test is actually about. Will delete it, thanks.